.circleci/config.yml

```diff
@@ -1,31 +1,53 @@
 version: 2.1
 
 orbs:
-  plugin-ci: mattermost/plugin-ci@0.1.0
+  plugin-ci: mattermost/plugin-ci@volatile
 
 workflows:
   version: 2
-  deploy:
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - master
     jobs:
       - plugin-ci/lint
       - plugin-ci/test
       - plugin-ci/build
-      - plugin-ci/deploy-release:
```

Comment:
`plugin-ci/deploy-release` should not be used any longer.

Reply:
Just curious, why?

Reply:
That was used to push the plugins to the S3 bucket to be used as prepackaged plugins. With `/mb cutplugin` in place, that is not longer needed.